ivy.el

```diff
@@ -104,6 +104,10 @@
   '((t :inherit dired-directory))
   "Face used by Ivy for highlighting subdirs in the alternatives.")
 
+(defface ivy-org
+  '((t :inherit org-level-4))
+  "Face used by Ivy for highlighting org files in the alternatives.")
```

Comment:
In what alternatives? How about something like "highlighting Org buffer names (or candidates) during completion" instead?

Reply:
I just referred to `Face used by Ivy for highlighting subdirs in the alternatives.` just above.

Improving all the other docstrings is outside the scope of that PR.

Reply:
> Improving all the other docstrings is outside the scope of that PR.

Sure, but that doesn't mean we should introduce docstrings that we think can be easily clarified.

Reply:
let's fix both at the same time in another PR.